ntp-daemon/src/system.rs

```diff
@@ -152,6 +160,7 @@ enum PeerStatus {
     NoMeasurement,
     /// This peer has sent snapshots taken in the current reset epoch. We store the most recent one
     Valid(PeerSnapshot),
+    NotFit(PeerSnapshot),
```

Comment:
Fitness is a time-dependent state, I think we should step away from encoding fitness in the peerstatus enum here and just check it upon use.

Reply:
like above, this does mean that an invalid snapshot can trigger a clock select. That is always correct, but could be inefficient.

Reply:
see above